client/post-editor/post-editor.jsx

```diff
@@ -69,6 +69,7 @@ import { isSitePreviewable } from 'state/sites/selectors';
 import { removep } from 'lib/formatting';
 import QuickSaveButtons from 'post-editor/editor-ground-control/quick-save-buttons';
 import EditorRevisionsDialog from 'post-editor/editor-revisions/dialog';
+import { regexp as shortCodeRegexp } from 'lib/shortcode';
```

Comment:
Please notice (here and in the other occurrences in this change) that "shortcode" does not have the capital C.
See: https://codex.wordpress.org/Shortcode_API